File: Java/src/Faccat/Exercicio6c.java
package Faccat;

import java.util.Scanner;

public class Exercicio6c {
    public static void main(String[] args) {
        double raio, pi, area;
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o valor do raio: ");
        raio = scanner.nextDouble();

        pi = Math.PI;
        area = pi * Math.pow(raio, 2);

        System.out.println("A área do círculo é: " + area);

        scanner.close();
    }
}

File: JavaScript/Faccat/Exercicio6c.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Digite o valor do raio: ', function(raio) {
  const pi = Math.PI;
  const area = pi * raio ** 2;
  console.log('A área do círculo é: ' + area);

  rl.close();
});